backend/src/stats.js

```diff
@@ -93,59 +106,78 @@ async function aggregateNewAccountsCountByDate() {
+    const genesisCount = await queryGenesisAccountCount();
+    if (
+      NEW_ACCOUNTS_COUNT_AGGREGATED_BY_DATE &&
+      DELETED_ACCOUNTS_COUNT_AGGREGATED_BY_DATE
+    ) {
+      const startDate = NEW_ACCOUNTS_COUNT_AGGREGATED_BY_DATE[0].date;
+      const dateArray = generateDateArray(startDate);
+      let changedAccountsCountByDate = dateArray.map((date) => ({
+        date: date,
+        accountsCount: 0,
+      }));
+      changedAccountsCountByDate[0].accountsCount = Number(genesisCount.count);
+
+      let newAccountMap = new Map();
+      for (let i = 0; i < NEW_ACCOUNTS_COUNT_AGGREGATED_BY_DATE.length; i++) {
+        newAccountMap.set(
+          NEW_ACCOUNTS_COUNT_AGGREGATED_BY_DATE[i].date,
+          Number(NEW_ACCOUNTS_COUNT_AGGREGATED_BY_DATE[i].accountsCount)
+        );
+      }
 
-    for (let i = 1; i < contractsRows.length; i++) {
-      if (contractsRows[i].date !== contractsRows[i - 1].date) {
-        cumulativeContractsDeployedByDate.push({
-          date: contractsRows[i - 1].date,
-          contractsCount: contractsDeployed.size,
-        });
+      let deletedAccountMap = new Map();
+      for (
+        let i = 0;
+        i < DELETED_ACCOUNTS_COUNT_AGGREGATED_BY_DATE.length;
+        i++
+      ) {
+        deletedAccountMap.set(
+          DELETED_ACCOUNTS_COUNT_AGGREGATED_BY_DATE[i].date,
+          Number(DELETED_ACCOUNTS_COUNT_AGGREGATED_BY_DATE[i].accountsCount)
+        );
       }
-      contractsDeployed.add(contractsRows[i].deployedContracts);
-    }
-    UNIQUE_DEPLOYED_CONTRACTS_COUNT_AGGREGATED_BY_DATE = cumulativeContractsDeployedByDate;
-    console.log("UNIQUE_DEPLOYED_CONTRACTS_COUNT_AGGREGATED_BY_DATE updated.");
-  } catch (error) {
-    console.log(error);
-  }
-}
 
-async function aggregateActiveContractsCountByDate() {
-  try {
-    const activeContractsCountByDate = await queryActiveContractsCountAggregatedByDate();
-    ACTIVE_CONTRACTS_COUNT_AGGREGATED_BY_DATE = activeContractsCountByDate.map(
-      ({ date: dateString, active_contracts_count_by_date }) => ({
-        date: formatDate(new Date(dateString)),
-        contractsCount: active_contracts_count_by_date,
-      })
-    );
-    console.log("ACTIVE_CONTRACTS_COUNT_AGGREGATED_BY_DATE updated.");
+      for (let i = 0; i < changedAccountsCountByDate.length; i++) {
+        if (newAccountMap.get(changedAccountsCountByDate[i].date)) {
+          changedAccountsCountByDate[i].accountsCount =
+            changedAccountsCountByDate[i].accountsCount +
+            newAccountMap.get(changedAccountsCountByDate[i].date);
+        }
+        if (deletedAccountMap.get(changedAccountsCountByDate[i].date)) {
+          changedAccountsCountByDate[i].accountsCount =
+            changedAccountsCountByDate[i].accountsCount -
+            deletedAccountMap.get(changedAccountsCountByDate[i].date);
+        }
+      }
+      LIVE_ACCOUNTS_COUNT_AGGREGATE_BY_DATE = changedAccountsCountByDate.reduce(
```

Comment:
Let's have a `cumulativeSumArray` method on the backend as well (it seems you initially copied it, but then removed it). This implementation is not reader-friendly, so it is better to have a name for it.